crawler/config_parser.py

```diff
@@ -20,6 +20,16 @@ def parse_crawler_config(config_path='crawler.conf'):
     vdt = Validator()
     _config.validate(vdt)
 
+    enabled_crawlers = _config['crawlers']
```

Comment:
This is doing exactly what validate() is supposed to do: merge the user configs with the defaults. Is there a way to keep using validate() and get the same result?